server/db/model/Event.js

```diff
@@ -1,4 +1,73 @@
 /* eslint-disable */
 var mongoose = require('../index.js'),
     config = require('../../../config/default.js'),
-    Schema = mongoose.Schema;
+    Schema = mongoose.Schema
+Location = require('./Location.js');
+
+// Define the document Schema
+var schema = new mongoose.Schema({
+    title: String,
+    desc: String,
+    startDate: {
+        type: Date,
+        default: Date.now()
+    },
+    endDate: {
+        type: Date,
+        default: Date.now()
+    },
+    category: {
+        type: String,
+        enum: [
+            'General',
+            'Food',
+            'Tech Talk',
+            'Sponsor Event',
+            'we should get a config list for this'
```

Comment:
@antoniomika what is the proper way to handle this mongo style? Or at least non-relational style